src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -279,25 +283,20 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseRep
      * @param complaintType the type of complaints we are interested in
      * @return the ResponseEntity with status 200 (OK) and a list of complaints. The list can be empty
      */
-    @GetMapping("/courses/{courseId}/complaints")
+    @GetMapping("courses/{courseId}/complaints")
```

Comment:
you could also remove "/" in line 240,258,278,315 but its just minor thing, i just added comment because you changed them for other endpoints....